libs/openchallenges/home/src/lib/home.component.html

```diff
@@ -1,4 +1,21 @@
 <main class="base mat-typography">
+  <div id="announcement">
+    <h4>
+      Welcome to
+      <a href="/about" target="_blank">OpenChallenges</a>!
+    </h4>
+    <h3>This website uses a preview of the OpenChallenges (OC) Database</h3>
+    <h4>
+      We are working hard to update the DB with more challenge data. The comprehensive DB will be
+      released in September or October 2023. We encourage organizations to
+      <a
+        href="https://docs.google.com/forms/d/e/1FAIpQLSdLZ6C5ZIzLZS147bmvNm2eJ3kq_88ecHS7ADjD4KaapGhO8g/viewform"
+        target="_blank"
+        >submit new challenges</a
+      >
+      and update existing ones to ensure that their challenge data are comprehensive.
```

Comment:
```suggestion
      and update existing ones to ensure that their challenge data are complete. To provide an update, go to your challenge profile page(s), then click on "Suggest an edit here!" at the bottom of the page.
```

Reply:
I'm afraid that this would make the message too long. I assume that if we tell users that they can edit existing data, they will go to where the existing data are (challenge profile page) and find the link to edit the data. I agree that some may not find the link but I would really try to keep the message in the banner short and clear.

Reply:
^ a daring assumption... 😆 But no, I definitely agree that this edit is too long.